libtelemetry/src/test/java/com/mapbox/android/telemetry/MapboxTelemetryTest.java

```diff
@@ -29,6 +32,9 @@
 
 public class MapboxTelemetryTest {
 
+  @Rule
+  public TemporaryFolder mTempFolder = new TemporaryFolder();
```

Comment:
why `mTempFolder`? should we be consistently not using hungarian notation even in tests?